violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/EditMenu.java

```diff
@@ -78,10 +78,9 @@ public void actionPerformed(ActionEvent event) {
                     IEditorPart activeEditorPart = getActiveEditorPart();
                     IEditorPartBehaviorManager behaviorManager = activeEditorPart.getBehaviorManager();
                     List<UndoRedoCompoundBehavior> found = behaviorManager.getBehaviors(UndoRedoCompoundBehavior.class);
-                    if (found.size() != 1) {
-                        return;
+                    if (found.size() >= 1) {
```

Comment:
hmm... shall I fix it? I haven't changed it because originally file had braced styled like this. I might change it if it creates inconsistency with rest of the project